### ISSUE TITLE
feat: accessibility updates for UI

### ISSUE THREAD

**Issue by FoxTheSin** [yearn/yearn-finance-v3#589]
**Elevator Pitch**
Some themes and titles are missing or inconsistent from objects for accessibility users (screen readers, etc.)

**Customer Value**
will benefit users who use screen readers (or potentially others I haven't thought of.)

**Definition of Done**
A screen reader could easily scroll through our site. Hopefully any accessibility user would be able to access our site.

**Design Approach**
Approach and considerations to build the task list / Existing code patterns in production code base that you will base your work off of.

**Additional Context**
Screenshots, UI/UX designs, links or other that will help implementing the feature.

**Comment by nuggetnchill** [comment]
Before I go any further, is it cool if I just use some placeholder copy for now?
Everything will be editable in common.js so we can have multiple language for screenreader as well

<img width="432" alt="image" src="https://user-images.githubusercontent.com/65918030/163518338-eb365646-dc2c-4356-8e5f-605e312f4a5e.png">

**Comment by pentcle** [comment]
hey @FoxTheSin didn't we re-write the on friday? going to find the card now

**Comment by FoxTheSin** [comment]
Removing this as it's a useless beast, breaking down into smaller sections (in linear, as I figure out how to piece apart into github will update issues there.)
